src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -126,7 +126,7 @@ protected function doDelete(array $ids)
     protected function doSave(array $values, $lifetime)
     {
         $ok = true;
-        $expiresAt = $lifetime ? time() + $lifetime : PHP_INT_MAX;
+        $expiresAt = time() + ($lifetime ?: 31557600); // 31557600s = 1 year
```

Comment:
Adding a constant creates a public interface change. Even if we can tag it as internal, it looks overkill and the comment looks good enough to me. Also, whenever we want to change this default duration, we won't have to create a new const.